static/app/views/insights/browser/resources/components/charts/resourceSummaryCharts.tsx

```diff
@@ -1,105 +1,23 @@
 import {Fragment} from 'react';
 
-import {t} from 'sentry/locale';
-import {MutableSearch} from 'sentry/utils/tokenizeSearch';
-import {Referrer} from 'sentry/views/insights/browser/resources/referrer';
-import {DATA_TYPE, FIELD_ALIASES} from 'sentry/views/insights/browser/resources/settings';
-import {useResourceModuleFilters} from 'sentry/views/insights/browser/resources/utils/useResourceFilters';
-import {InsightsLineChartWidget} from 'sentry/views/insights/common/components/insightsLineChartWidget';
 import * as ModuleLayout from 'sentry/views/insights/common/components/moduleLayout';
-import {useSpanMetricsSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
-import {
-  getDurationChartTitle,
-  getThroughputChartTitle,
-} from 'sentry/views/insights/common/views/spans/types';
-import {SpanMetricsField} from 'sentry/views/insights/types';
-
-const {
-  SPAN_SELF_TIME,
-  HTTP_RESPONSE_CONTENT_LENGTH,
-  HTTP_DECODED_RESPONSE_CONTENT_LENGTH,
-  HTTP_RESPONSE_TRANSFER_SIZE,
-  RESOURCE_RENDER_BLOCKING_STATUS,
-} = SpanMetricsField;
-
-function ResourceSummaryCharts(props: {groupId: string}) {
-  const filters = useResourceModuleFilters();
-
-  const mutableSearch = MutableSearch.fromQueryObject({
-    'span.group': props.groupId,
-    ...(filters[RESOURCE_RENDER_BLOCKING_STATUS]
-      ? {
-          [RESOURCE_RENDER_BLOCKING_STATUS]: filters[RESOURCE_RENDER_BLOCKING_STATUS],
-        }
-      : {}),
-    ...(filters[SpanMetricsField.USER_GEO_SUBREGION]
-      ? {
-          [SpanMetricsField.USER_GEO_SUBREGION]: `[${filters[SpanMetricsField.USER_GEO_SUBREGION].join(',')}]`,
-        }
-      : {}),
-  });
-
-  const {
-    data: spanMetricsSeriesData,
-    isPending: areSpanMetricsSeriesLoading,
-    error: spanMetricsSeriesError,
-  } = useSpanMetricsSeries(
-    {
-      search: mutableSearch,
-      yAxis: [
-        `epm()`,
-        `avg(${SPAN_SELF_TIME})`,
-        `avg(${HTTP_RESPONSE_CONTENT_LENGTH})`,
-        `avg(${HTTP_DECODED_RESPONSE_CONTENT_LENGTH})`,
-        `avg(${HTTP_RESPONSE_TRANSFER_SIZE})`,
-      ],
-      enabled: Boolean(props.groupId),
-      transformAliasToInputFormat: true,
-    },
-    Referrer.RESOURCE_SUMMARY_CHARTS
-  );
-
-  if (spanMetricsSeriesData) {
-    spanMetricsSeriesData[`avg(${HTTP_RESPONSE_TRANSFER_SIZE})`].lineStyle = {
-      type: 'dashed',
-    };
-    spanMetricsSeriesData[`avg(${HTTP_DECODED_RESPONSE_CONTENT_LENGTH})`].lineStyle = {
-      type: 'dashed',
-    };
-  }
```

Comment:
All 3 charts shared the data fetching code, but in [the new charts](https://github.com/getsentry/sentry/pull/89612/files#diff-a09e77212e8b79b7832e5832a92dd583e35a329a8a4ff6203137f7f2cd5ff381R23-R30) only the AverageSeries chart has this code since it is the only chart that uses these series (IIUC).